ci/scripts/github_skipped_tests_comment.py

```diff
@@ -195,10 +221,43 @@ def get_skipped_tests_comment(
     if len(skipped_list) == 0:
         logging.info("No skipped tests found.")
 
+    if not is_dry_run:
+        current_file = abspath(inspect.getsourcefile(lambda: 0))
+        additional_tests_to_check_file = (
```

Comment:
nit: use pathlib to clean stuff up, e.g. `Path(current_file).parent / "required_tests_to_run.json"`